docs/src/components/PropTable.js

```diff
@@ -143,7 +143,6 @@ export default function PropTable({ props: properties, Component }: Props) {
                                 const elem = document.getElementById(href);
                                 if (elem) {
                                   elem.scrollIntoView({
-                                    behavior: 'smooth',
```

Comment:
Removing this since it took too long to scroll down to the appropriate section of the page